src/lib/swapper/swappers/OsmosisSwapper/utils/constants.ts

```diff
@@ -1,7 +1,10 @@
+import { adapters } from '@shapeshiftoss/caip'
 import { SwapperName } from 'lib/swapper/api'
 
 export const DEFAULT_SOURCE = [{ name: SwapperName.Osmosis, proportion: '1' }]
 export const MAX_SWAPPER_SELL = '100000000000000000000000000'
 export const OSMOSIS_PRECISION = 6
 export const COSMO_OSMO_CHANNEL = 'channel-141'
 export const OSMO_COSMO_CHANNEL = 'channel-0'
+
+export const atomOnOsmosisAssetId = adapters.osmosisToAssetId('ATOM') ?? ''
```

Comment:
Make this a literal, `constants` file should contain constants - this makes it unable to grep for it.

This also loses the `AssetId` type narrowing because of opt chain - we shouldn't need an adapter to get a constant.